### PR TITLE
Tx2 blog prototype

### PR DESCRIPTION
**Pull request by HelmerDenDekker**
Prototype of the blog page ready for testing.
It has the Muggenhoeve-only and should:
- Show article
- Display the right header
- Show Fotogalerij header-only, no image-gallery included.

### CODE REVIEW

**Comment by HelmerDenDekker** [comment]
Closes #2 
Closes #3

File: vite.config.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+﻿// vite.config.js
+import { defineConfig } from 'vite'
+export default defineConfig({
```
Comment (TeamXerbutri) [inline vite.config.js:3]: Can be removed

File: public/data/xerbutri/txaue/txaue.html
Comment (TeamXerbutri) [inline public/data/xerbutri/txaue/txaue.html:1]: remove this file

File: public/data/xerbutri/txatx/txatx.html
Comment (TeamXerbutri) [inline public/data/xerbutri/txatx/txatx.html:1]: remove

File: public/data/spoor/l126/l126.html
Comment (TeamXerbutri) [inline public/data/spoor/l126/l126.html:1]: remove

File: public/data/gebouw/vbmh/nl_vbmh.txt
Comment (TeamXerbutri) [inline public/data/gebouw/vbmh/nl_vbmh.txt:1]: remove

File: public/data/gebouw/skfe/nl_skfe.html
Comment (TeamXerbutri) [inline public/data/gebouw/skfe/nl_skfe.html:1]: remove

File: public/data/gebouw/kwh/nl_kwh.html
Comment (TeamXerbutri) [inline public/data/gebouw/kwh/nl_kwh.html:1]: remove

File: components/router.js
Comment (TeamXerbutri) [inline components/router.js:1]: check todo's

File: components/adventure.js
Comment (TeamXerbutri) [inline components/adventure.js:1]: is this still relevant?

File: avontuur/style.css
Comment (TeamXerbutri) [inline avontuur/style.css:1]: remove?

File: avontuur/index.html
Comment (TeamXerbutri) [inline avontuur/index.html:1]: remove?

File: avontuur/adventure.js
Comment (TeamXerbutri) [inline avontuur/adventure.js:1]: remove?

File: assets/js/home.js
Comment (TeamXerbutri) [inline assets/js/home.js:1]: clean up

File: assets/css/objectcard.css [hunk 2]
```diff
@@ -0,0 +1,184 @@
+﻿@media only screen and (max-width: 756px) {
+	.tile:nth-child(-n+3) {
+		position: relative;
+		overflow: hidden;
+		display: block;
+		height: 200px;
+		background-color: #181818;
+		margin: 2px
```
Comment (TeamXerbutri) [inline assets/css/objectcard.css:8]: cleanup

File: assets/css/index.css [hunk 3]
```diff
@@ -1 +1,124 @@
...
\ No newline at end of file
+@media only screen and (max-width: 756px) {
+	body {
+		background-color: #404040;;
+		font-family: Verdana, Tahoma, sans-serif;
+		font-size: 100%;
+		margin: 0
```
Comment (TeamXerbutri) [inline assets/css/index.css:6]: cleanup

File: assets/css/adventure.css
Comment (TeamXerbutri) [inline assets/css/adventure.css:1]: cleanup

File: README.md
Comment (TeamXerbutri) [inline README.md:1]: Check and enhance

File: .documentation/adr/0004-routing.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+It has 3 templates/controllers for the website:
+
+- Home
+- Avontuur (Blog)
+- Maps
+
+The backend logic constructs the pages from a data model it gets from the database.
+
+It has a nice separation of concerns, returning html pages with the data from the model.
+The current Js is just for small interactions.
```
Comment (TeamXerbutri) [inline .documentation/adr/0004-routing.md:19]: javascript

**Review by TeamXerbutri — CHANGES_REQUESTED** [review]

File: vite.config.js (hunk 1 above)
Comment (HelmerDenDekker) [reply vite.config.js]: Removed

File: public/data/xerbutri/txaue/txaue.html
Comment (HelmerDenDekker) [reply public/data/xerbutri/txaue/txaue.html:1]: Replaced by working json

File: public/data/xerbutri/txatx/txatx.html
Comment (HelmerDenDekker) [reply public/data/xerbutri/txatx/txatx.html:1]: Replaced by working json

File: public/data/spoor/l126/l126.html
Comment (HelmerDenDekker) [reply public/data/spoor/l126/l126.html:1]: replaced by blog.json

**Review by TeamXerbutri — APPROVED** [review]

**Pull request merged**